master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -79,6 +79,10 @@ private[celeborn] class Master(
   private val authEnabled = conf.authEnabled
   private val secretRegistry = new MasterSecretRegistryImpl()
 
+  if (conf.logMasterConfEnabled) {
+    logInfo(getConf)
```

Comment:
IMO, `getConf` returns all config options of master and worker. But this should only log the master configs according to the name of `celeborn.master.logConf`. Right?

Reply:
I think with this [comment](https://github.com/apache/celeborn/pull/2442#discussion_r1554796260), we can just log everything in the CelebornConf for that master or worker.

Reply:
The getConf would only return whatever is put on the CelebornConf for that service, so it should just be master or worker configs.